guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -3,16 +3,44 @@
 
 In {Project}, you can use the package dependency resolution feature to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
 
-You can select to resolve package dependencies for any Content View that you want, or you can change the default setting to enable or disable resolving package dependencies for all new Content Views.
+You can select to resolve package dependencies per Content View or have it enabled by default via Settings -> Content -> "content view Dependency Solving Default".
+Enabling content view dependency solving by default is unrecommended unless the pitfalls (discussed later) are well-understood.
 
 Note that resolving package dependencies can cause significant delays to Content View promotion.
-The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View or solve dependencies across repositories.
+The more content and repositories in the content view, the greater the delay.
```

Comment:
It's a bit weird to say pitfalls will be discussed later and then mention a pitfall in the very next sentence